lib/trailblazer/finder/utils/string.rb

```diff
@@ -37,10 +37,11 @@ def self.to_date(value)
 
         def self.date?(date)
           return false unless
-              date.kind_of?(::DateTime) ||
-              date.kind_of?(::Date) ||
-              date.kind_of?(::String)
-          return false if date.kind_of?(::String) && date.size == 36 # Ignore uuids that could get casted to dates
+              date.is_a?(::DateTime) ||
```

Comment:
Does this also work if the `date` is a subclass instance of `DateTime`?

Reply:
yeah.